exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/cloud/reuters-edml.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schemas.exasol.com/edml-1.2.0.json",
+  "source": "reuters-out/reuters1-*.json",
+  "destinationTable": "NEWS",
+  "addSourceReferenceColumn": true,
+  "mapping": {
+    "fields": {
+      "id": {
+        "toVarcharMapping": {
+          "destinationName": "NEWS_ID",
+          "varcharColumnSize": 50
+        }
+      },
+      "date": {
+        "toVarcharMapping": {
+          "varcharColumnSize": 50
+        }
+      },
+      "title": {
+        "toVarcharMapping": {
+          "varcharColumnSize": 255,
+          "overflowBehaviour": "TRUNCATE"
+        }
+      },
+      "body": {
+        "toVarcharMapping": {
+          "varcharColumnSize": 1024,
+          "overflowBehaviour": "TRUNCATE"
+        }
```

Comment:
It would be good if we could also have a list mapping in here. The keys places and topics for example.

Reply:
we should then also add a example query to show, how to use the mapping.

Reply:
added